### ISSUE TITLE
Box SDK’s auth WebView circumvents web filters

### ISSUE THREAD

**Issue by alenz316** [box/box-android-sdk#451]
Hey Awesome People,

Our Android app uses the Box SDK and is available to EDU (Education) ChromeOS users. A client of ours submitted a ticket that demonstrated that students were able to circumvent web filters by using the Box SDK’s auth WebView to navigate to blocked sites. Details and a solution can be found at https://chromeos.dev/en/education/improving-webview-compatibility#launch-external-sites-in-the-browser

**In short, the problem would be solved if the Box SDK could provide a mechanism to redirect OAuth to use either the system browser app or a Chrome Custom Tab.**

**Comment by lukaszsocha2** [comment]
Hi @alenz316,
unfortunately Android SDK is not currently actively supported by Box. It is planned to return to `maintained` state in some as yet unknown future. However you are welcome to prepare your own pr with proposed change - we can review it and make a release.
Best @lukaszsocha2
